src/angular/app.js

```diff
@@ -212,8 +212,8 @@ controllers.controller('ResultsCtrl', ['$scope', 'Search', function ($scope, Sea
 
     // gets the activity details of the service 
     $scope.getActivityDetails = function(result){
-        //define a variable to store the 'closing time'
-        var activityDetails = null;
+        // Define a default variable value for activity details
+        var activityDetails = ["Unknown"];
```

Comment:
Let's not worry about translations right now, that's a whole different can of worms.